### DISCUSSION TITLE
separate hostname from application name

### DISCUSSION THREAD

**Discussion by jniggemann** [tteck/Proxmox#77] [Ideas]
Hi, I wanted to give the pihole setup script a try, but I already have one running under the hostname `pihole` so that failed, because the hostname is derived from the application name.

Are there plans to make the hostname configurable?

**Comment by tteck** [comment]
After successful script completion, click on the LXC then click on DNS to change hostname

**Reply by jniggemann** [reply]
Strange, I had it fail at create_lxc with a message similar to "hostname already present in container XXX" or sth like that.

**Reply by tteck** [reply]
Shouldn't there's no hostname check

**Reply by jniggemann** [reply]
't was a proxmox message, but I can't reproduce it, so possibly some error / mistake on my side.
